scripts/ceph-windows/setup_libvirt_ubuntu_vm

```diff
@@ -7,7 +7,7 @@ if [[ -z $LIBVIRT_DIR ]]; then echo "ERROR: The env variable LIBVIRT_DIR is not
 export UBUNTU_VM_IMAGE_URL=${UBUNTU_VM_IMAGE_URL:-"https://cloud-images.ubuntu.com/minimal/releases/jammy/release/ubuntu-22.04-minimal-cloudimg-amd64.img"}
 export UBUNTU_VM_NAME=${UBUNTU_VM_NAME:-"ceph-ubuntu-vstart-${JOB_NAME}-${BUILD_ID}"}
 export UBUNTU_VM_VCPUS="$(nproc)"  # use all the CPU cores avaiable on the host
-export UBUNTU_VM_MEMORY="65536"    # 64 GB
+export UBUNTU_VM_MEMORY=${UBUNTU_VM_MEMORY:-"16384"}  # 16 GB
```

Comment:
> Since the Windows job no longer uses memstore, we can probably reduce the Linux vm memory to 16GB.

It still builds Ceph in that VM before spawning a vstart cluster, right?  For a regular vstart cluster with 3 OSDs, 16G is probably on the high side, but the build is an entirely different story.

We should really figure out a way to build Ceph just once and share the artifacts for jobs that are a must for every PR: `make check`, `api`, `windows`, etc.

Reply:
> It still builds Ceph in that VM before spawning a vstart cluster, right? For a regular vstart cluster with 3 OSDs, 16G is probably on the high side, but the build is an entirely different story.

Exactly, here's the full sequence: https://github.com/ceph/ceph-build/blob/d67fe99f91eea5e50a00844e3b54880619de7c20/ceph-windows-pull-requests/config/definitions/ceph-windows-pull-requests.yml#L70-L79.

> Makes sense, although that would require the artifacts to be stored on a shared environment and I have very limited access and knowledge of the Ceph CI infra. For now, all I can do is adjust the VM memory and/or filter nodes by labels.

We should really figure out a way to build Ceph just once and share the artifacts forbetween jobs that are a must for every PR: make check, api, windows, etc.

Sorry for editing your comment btw, I mean to leave a reply :).

Reply:
> Makes sense, although that would require the artifacts to be stored on a shared environment and I have very limited access and knowledge of the Ceph CI infra. For now, all I can do is adjust the VM memory and/or filter nodes by labels.

Well, I don't see anything wrong with trying to use less memory as is.  Let's see how it goes.